### DISCUSSION TITLE
Use rems for shadows (and potentially also for thicker border widths)

### DISCUSSION THREAD

**Discussion by danielberndt** [tailwindlabs/tailwindcss#834] [Ideas]
Hi,

I just noted that most of your default values are defined using `rem` of which I am a big fan.

From what I've seen there are two notable exceptions to this though

- shadows https://github.com/tailwindcss/tailwindcss/blob/a7a95b15c85c0c8d9875240de9e2e94f05789ec0/defaultConfig.stub.js#L797-L804

- and border-widths
https://github.com/tailwindcss/tailwindcss/blob/a7a95b15c85c0c8d9875240de9e2e94f05789ec0/defaultConfig.stub.js#L432-L438

Especially for shadows it would be really nice if they would scale along with all the other `rem` defined properties like widths and spacings.

For borders it's somewhat debatable. I definitely would leave `1px` but for thicker borders it might also be beneficial to use something `rem` based. But wouldn't mind offloading this specific decision to the user.

Curious to hear your thoughts on the shadows!

**Comment by adamwathan** [comment]
Yeah this is a good point and worth considering! Agree the 1px border makes sense as px to avoid weird half pixel crap, but maybe the others should be in rem, and the shadows probably should be in rem too. Going to keep this open until I have a chance to think about it more.
